ansible/vars/common.yml

```diff
@@ -82,15 +82,21 @@ harvest_sources: false
 ckan_version: "2.8.0"
 ckan_branch: "ckan-{{ ckan_version }}"
 
-email_domain: "localhost"
-email_domain_access: "localhost"
+email_domain: "dev.avoindata.fi"
```

Comment:
Email domain should probably be variable based on the ansible inventory used (enviroment). There is already `email_domain` var in ansible/vars/common.yml that is probably the correct value for this.